en/lesson-01.md

```diff
@@ -0,0 +1,17 @@
+---
+title: What is LaTeX and how does it work?
+---
+
+## How does LaTeX work?
+
+Unlike common word processors such as Microsoft Word or LibreOffice, LaTeX usually does not provide WYSIWYG. With LaTeX one takes a good text editor, writes the text and enriches the text with markup. This markup tells LaTeX about the logical meaning of certain elements of the text, similar to the way HTML does.
+
+Take for example a `<h2>` headline indicating a new section in an HTML-document. LaTeX also has a command for this, here one would use the `\section` command.
+
+## From here on
+
+For more information on the history of TeX and LaTeX see:
+
+*
```

Comment:
what is this `*`doing?

Reply:
I don't know, I didn't add it. I merged with master because the filenames changed, I guess it was added there.

Reply:
@FrankMittelbach can confirm that this was added to this branch in e3861e2 which was the merge of master into this branch. Didn't search for when it was added to master though. I'd guess that the author wanted to add bullet points there with other resources and used this as a placeholder.